pkg/spec/spec.go

```diff
@@ -306,6 +306,11 @@ func CreateConfigToOCISpec(config *CreateConfig) (*spec.Spec, error) { //nolint
 	if err := config.GetVolumesFrom(); err != nil {
 		return nil, errors.Wrap(err, "error getting volume mounts from --volumes-from flag")
 	}
+	fsMounts, err := config.GetFSMounts(configSpec.Mounts)
```

Comment:
Do we want this to supercede mounts given by `-v` in precedence? I think we might want it to, in which case it should be moved below `GetVolumeMounts`

Also, we need to handle overlapping mounts already in the spec, as we do for the mounts returned by `GetVolumeMounts`

Reply:
So here I understand you mean, `specs mount`  <  `-v mount` < `--mount`
i.e. any mount point defined in specs is superceded by `-v` mounts.
&    mount point defined by `-v` mount  will be superceded by ``-mount`` option